u4-databases/views/uni-iteration-details.hbs

```diff
@@ -0,0 +1,13 @@
+{{#>uni iteration}}
```

Comment:
เลย์เอาต์ `uni` ไม่มีพารามิเตอร์ใดๆ `iteration` ไม่สมเหตุสมผลที่นี่ คุณควรเขียนเท่านั้น:
The `uni` layout doesn't have any parameters. `iteration` does not make sense here. You should only write:
```js
{{#>uni }}
```
สำหรับการอ้างอิง หาก `uni` มีพารามิเตอร์ (เวอร์ชันนี้ไม่มี) คุณจะต้องตั้งค่าดังนี้:
For reference, if `uni` did have a parameter (it doesn't), you would need to set its value like this:
```js
{{#>uni iteration=someValue}}
```